src/python/pants/backend/javascript/target_types.py

```diff
@@ -90,6 +92,12 @@ class JSTestExtraEnvVarsField(TestExtraEnvVarsField):
     pass
 
 
+class JSTestBatchCompatibilityTagField(TestsBatchCompatibilityTagField):
+    help = help_text(
```

Comment:
I sort of think that setting `default="default"` here would be a sane default, but I also believe that there's value in keeping this as close to 1:1 as the python variant.